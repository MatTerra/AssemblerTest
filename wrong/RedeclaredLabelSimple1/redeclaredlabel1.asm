section DATa
just_single_data: cONSt 1
section TExt
adding_one: add just_single_data
ADDING_one: add just_single_data
;
; This label was redefined
stop
