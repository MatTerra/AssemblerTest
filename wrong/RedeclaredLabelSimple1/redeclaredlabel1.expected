Line 4/5 Label adding_one redefined - Semantic
